rfcs/Base256Emoji.md

```diff
@@ -55,6 +55,8 @@ Emoji|UTF Code|Byte Value
 -+-+-
 ☉|U+2609|16
 -+-+-
+🌠|U+1F320|16
+-+-+-
```

Comment:
I get that this isn't a particularly serious encoding, but why would we make it intentionally confusing by having two encodings here one of which isn't an emoji?

As for the emoji? Sure, it'd probably be nice if our emojis were more distinct (e.g. who doesn't like a 🐉) but have another (4th?) star emoji is fine I guess 🤷.

Reply:
> I get that this isn't a particularly serious encoding, but why would we make it intentionally confusing by having two encodings here one of which isn't an emoji?

~~B_a_***ckWA***R*d CO*mP*ATib*il***iTY***~~

Because that makes the fun of implementing it longer.
It being kinda hard to implement was always one of my goals. (that why the variable sized codepoints for example)

> As for the emoji? Sure, it'd probably be nice if our emojis were more distinct (e.g. who doesn't like a dragon) but haveing another (4th?) star emoji is fine I guess shrug.

Fair, but sounds like comments that should have been made for the 5+ previous month #88 was in review.

As of why, I just wanted to fit the space theme.

Reply:
Update: Here there be dragons.

Reply:
> It being kinda hard to implement was always one of my goals

That doesn't really compute for me. Why make things harder when they could be easier at no cost?

> Fair, but sounds like comments that should have been made ... As of why, I just wanted to fit the space theme.

ya, fair enough. I don't feel particularly strong about this.

Reply:
> Update: Here there be dragons.

Ok, actually LOL at the dragon emoji being the one with the compat thing. I still think having the compat thing is kind of nuts, but having seen the change I think it's pretty funny 🤣

Reply:
(FYI that a stelaris (paradox's space XXXX RTS) reference, Here there be dragons is the name of a DLC with rogue powerful giant dragons granting bonuses when defeated)

Reply:
the table seems to be broken:
<img width="797" alt="image" src="https://user-images.githubusercontent.com/356110/173435541-db61bccb-27ac-4291-a5c6-0aea32d2865d.png">

Reply:
@ribasushi thx fixed!